config/rbac/trusteeconfig_editor_role.yaml

```diff
@@ -0,0 +1,31 @@
+# permissions for end users to edit trusteeconfigs.
```

Comment:
Was this rbac generated by SDK ?

Reply:
not sure but it is aligned with the KbsConfig handling